go_cli.py

```diff
@@ -54,6 +58,13 @@
 def generate_random_uid():
     return str(uuid.uuid4())
 
+def load_config():
```

Comment:
We need to handle this carefully. 
1. All the user info including user id should be stored in this config. 
2. But at the same time, we should be backward compatible. If a user updates this system, then we should not assign them a new user-id and carry forward their previous user-id.